_posts/2020-02-10-hhvm-4.44.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "HHVM 4.44"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.44 is released! This release marks the end of support for 4.37;
+HHVM 4.38&ndash;4.43 remain supported, as do the 4.8 and 4.32 LTS releases.
+
+# Breaking Changes
+
+- the `__Deprecated` attribute is no longer permitted on any types, such as
+  classes, enums, or type aliases; it had no effect in these positions.
+- `(int) NAN` is now consistently the minimum 64-bit signed integer, instead of
+  0. In previous releases, this behavior could differ at runtime compared to
+  when constant-folding.
+- square bracket literals for PHP arrays (e.g. `['foo', 'bar']`) are now a
+  parse error. `array()` is still supported, but will be removed in the
+  near future. Ideally, use `vec`/`dict`/`keyset` instead, but `varray` and
+  `darray` remain supported.
+
+# Future Changes
+
+- comparison operations (e.g. `<`, `<=`, `>`, `cmp`, `<=>` etc) will be changed
```

Comment:
Does PHP / Hack have a `cmp` operator?
It is not a function `\cmp()` and `\HH\cmp()` are not defined.
`=[1]cmp[2]` is not valid in the debugger.

Guessing from context this was meant to be `=>`.

Reply:
will remove, probably variant_cmp()